src/app/app.component.ts

```diff
@@ -49,10 +49,17 @@ export class AppComponent implements AfterViewInit {
   }
 
   ngAfterViewInit(): void {
+
+    /**
+    * Note: If using an input like [triggers]="[ '.application-menu-trigger' ]"
+    * hookup the app menu trigger once the afterViewInit is called. This will
+    * ensure that the toolbar has had a chance to create the application-menu-trugger
+    * button.
+    * this.applicationMenu.triggers = [ '.application-menu-trigger' ];
```

Comment:
Probably want this in a time out so it can complete a tick.

And the next set of code in another timeout so it can rely on the button being hooked up

Reply:
I tested it and it works fine, but its just a comment now, as its not the default way to do it.

Reply:
Thought maybe was a good place to put it  but maybe we dont  need?

Reply:
Ok, what is the default way to do it?

Reply:
Without adding the triggers input inline on the component.